roles/jwflory.teleirc/tasks/compile.yml

```diff
@@ -9,22 +9,25 @@
 - name: git clone/pull RITlug/teleirc
   git:
     repo: "https://github.com/RITlug/teleirc.git"
-    dest: "/tmp/teleirc"
-    version: "{{ default_version }}"
+    dest: "/tmp/teleirc/{{ item.value.cn }}"
```

Comment:
Does `/tmp/teleirc/` need to be created first?

Reply:
I believe it creates `/tmp/teleirc/` if it doesn't exist yet (pretty sure I blew away that directory during testing).